src/frontend-pwa/src/App.tsx

```diff
@@ -5,23 +5,25 @@ import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
 import ViewRouter from './routes/ViewRouter';
 import SplashScreen from './components/SplashScreen/SplashScreen';
+import LocationsArray from './Type/LocationsArray';
 
 function App() {
   const [isLoading, setIsLoading] = useState(true);
+  const [locationData, setLocationData] = useState<LocationsArray>([]);
```

Comment:
Does this throw an error at runtime due to being empty?

Reply:
@Bottle7 Not in any of my tests

Reply:
@LocalNewsTV Just tried my branch and no issues on his side either